pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -98,12 +98,6 @@ func runE(logger log.Logger, streams cmd.IOStreams, flags *flagpole) error {
 		cluster.CreateWithDisplayUsage(true),
 		cluster.CreateWithDisplaySalutation(true),
 	); err != nil {
-		if errs := errors.Errors(err); errs != nil {
```

Comment:
we have more suitable logging for this in main:
![image](https://user-images.githubusercontent.com/917931/80275236-dd0b2b80-8694-11ea-92c5-6b774a2473a5.png)